mindmeld/active_learning/plot_manager.py

```diff
@@ -449,6 +434,71 @@ def plot_all_epochs(
                 fig.savefig(self.get_img_path(y_keys, title))
                 self.plt.clf()
 
+    def plot_all_epochs(
+        self,
+        y_keys: List,
+        display: bool = False,
+        save: bool = True,
+        use_aggregate_statistic: bool = True,
+    ):
+        """Plot all epochs. Creates a plot for each strategy.
+        Args:
+            y_keys (list): Keys to access the data from a epoch to be used as y values for plotting.
+            display (bool): Whether to show the plot.
+            save (bool): Whether to save the plot.
+            use_aggregate_statistic (bool): If True, the aggregate_statistic will be used as the
+                label for the y_axis. If False, the class_level_statistic will be used.
+        """
+        self.create_plot_dir(y_keys)
+        key = 1 if "entities" in y_keys else 0
+        strategies = self.strategies[key]
+        if strategies:
+            for strategy in strategies:
+                n_epochs = len(self.accuracies_data[key][strategy].keys())
+                all_y_values = []
+                for epoch in range(n_epochs):
+                    epoch_dict = self.accuracies_data[key][strategy][str(epoch)]
+                    x_values = PlotManager.get_across_iterations(
+                        epoch_dict, ["num_sampled"]
+                    )
+                    y_values = PlotManager.get_across_iterations(
+                        epoch_dict, ["accuracies"] + y_keys
+                    )
+                    self.plt.plot(x_values, y_values)
+                    all_y_values.append(y_values)
+                max_len = max([len(i) for i in all_y_values])
+                for y_values in all_y_values:
+                    y_values = np.pad(
```

Comment:
Nit: From my understanding, this is padding by extending the last value. A comment explaining that may be helpful here